pom.xml

```diff
@@ -31,6 +31,7 @@
 		<module>spring-cloud-gcp-starters</module>
 		<module>spring-integration-gcp</module>
 		<module>spring-cloud-gcp-docs</module>
+		<module>spring-cloud-gcp-stream</module>
```

Comment:
if this is a  pubsub specific binder, why not reflect that in the artifact name? such as ` spring-cloud-stream-binder-pubsub` or similar? `spring-cloud-gcp-stream` doesn't give the impression that it is a binder implementation.

Reply:
The `spring-cloud-gcp-stream` module contains two sub-modules: `spring-cloud-gcp-stream-binder` and `spring-cloud-gcp-stream-core`.

I agree this warrants a rename if we're planning other binders besides Pub/Sub. If Pub/Sub is going to be our only binder, then perhaps naming is good as-is?